server/applicationPattern/applicationPattern/onfModel/services/LogicalTerminationPointWithMappingServices.js

```diff
@@ -450,10 +450,10 @@ function findAndUpdateLogicalTerminationPointApplicationAndReleaseInstanceGroupA
                 operationNamesByAttributes,
                 operationsMapping
             );
-            httpClientConfigurationStatus = await updateHttpClientInterface(
-                httpClientUuid,
-                releaseNumber,
-            )
+            for(let operationClientConfigurationIndex = 0; operationClientConfigurationIndex < operationClientConfigurationStatusList.length; operationClientConfigurationIndex++){
```

Comment:
Running a loop here would replace the httpclientInterface for every run . It is not a better idea maybe we can check the alternative as discussed.